library/styleguideComponents/Examples/ExamplesRenderer.js

```diff
@@ -0,0 +1,31 @@
+import 'core-js/modules/es.function.name';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'rsg-components/Styled';
+
+var styles = function styles() {
+    return {
+        // Just default jss-isolate rules
+        root: {},
+    };
+};
+
+export function ExamplesRenderer(_ref) {
+    var classes = _ref.classes,
+        name = _ref.name,
+        children = _ref.children;
+    return React.createElement(
+        'article',
+        {
+            className: classes.root,
+            'data-testid': name + '-examples',
+        },
+        children,
+    );
+}
```

Comment:
can we just use the original ExamplesRenderer from react styleguidist. If not why not use jsx instead of React.createElement?